Jenkinsfile

```diff
@@ -159,22 +115,12 @@ pipeline {
                   /**
                   Checkout the code and stash it, to use it on other stages.
                   */
-                  stage('Install .Net SDK') {
+                  stage('Install tools') {
                     steps {
                       deleteDir()
+                      unstash 'source'
                       dir("${HOME}"){
```

Comment:
```suggestion
                      dir("${HOME}/${BASE_DIR}"){
```

Reply:
It's required to run the script within the HOME folder to be able to install the tools in the right folder which later on are added to the PATH env variable. See https://github.com/elastic/apm-agent-dotnet/pull/232/files#diff-0f0d09b1cb72e375ba048ed90acbe068R10 which refers to the relative path where the script is executed from.

Reply:
ok